### DISCUSSION TITLE
Action fails for self-hosted windows runner

### DISCUSSION THREAD

**Discussion by matriculus** [subosito/flutter-action#223] [General]
![image](https://user-images.githubusercontent.com/10431830/233828317-6200b80e-9960-4ad3-9da7-611e4d9ed627.png)

I get this error and the github action fails.
Says I do not have WSL. But  WSL is installed and it was running the whole time when this action was performed.
Action runs successfully for Github hosted windows-latest env.

**Comment by prinkeeras** [comment]
any fix for this

**Comment by matriculus** [comment]
Is there any fix on this? I need it desperately. As of now, this action does not work in self hosted windows runner.
